Dockerfile

```diff
@@ -16,19 +29,13 @@ RUN yum install -y \
   postfix \
   krb5-libs \
   krb5-workstation \
-  python3-{devel,pip,pygit2}
-
-ADD http://download.devel.redhat.com/rel-eng/RCMTOOLS/rcm-tools-rhel-8-baseos.repo /etc/yum.repos.d/
-# rhpkd didn't in default repo, and to use extra repo need switch off ssl check
-RUN echo "sslverify=false" >> /etc/yum.conf; yum install -y rhpkg
-
-# Those environment variables are required to install pycurl, koji, and rpkg with pip
-ENV PYCURL_SSL_LIBRARY=openssl RPM_PY_SYS=true
-
-RUN pip3 install koji tox twine setuptools wheel codecov future \
-  rh-doozer==1.2.31 rh-elliott==1.0.18 rh-ocp-build-data-validator==0.1.24
+  python3-{devel,pip,pygit2} rhpkg && \
+  yum autoremove && yum clean all && \
+  PYCURL_SSL_LIBRARY=openssl RPM_PY_SYS=true pip3 --no-cache-dir install \
+  koji tox twine setuptools wheel codecov future \
+  rh-doozer==1.2.31 rh-elliott==1.0.18 rh-ocp-build-data-validator==0.1.24 && \
```

Comment:
I think it is better to remove the hardcoded version or define them as build arguments, otherwise we'll need to remember to bump them all the time.

Reply:
I copied from previous version. I guess it want to start with  a non-root user, although I don't know why, maybe it's for use with tox, since the config file didn't in this repo.